LunaMary/src/Quiz.java

```diff
@@ -0,0 +1,23 @@
+import java.util.List;
+
+
+class Quiz {
+
+    private List<IQuestion> iQuestions;
+    private TypeQuiz typeQuiz;
+//    private String ageRange;
+
+    public Quiz(List<IQuestion> iQuestions, TypeQuiz typeQuiz) {
```

Comment:
You should have two constructors one that receives a list of questions and one that doesn't.
So you can have methods to add questions to the Quiz, so it will be more flexible.